### PR TITLE
Fixed the Hero Section

### PR DESCRIPTION
**Pull request by Shubham-1105**

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -294,6 +295,13 @@ input:checked + label:after {
   justify-content: center;
   align-items: center;
 }
+@media screen and (max-width:500px){
```
Comment (saviourcode) [inline style.css:298]: The current 100vh for all screen width is okay. Don't reduce it.

**Comment by saviourcode** [comment]
Do that small change. Or else it looks fine.

**Comment by Shubham-1105** [comment]
Which change?

**Comment by saviourcode** [comment]
Check the review

**Comment by Shubham-1105** [comment]
review it is not showing

**Review by sjchoure — CHANGES_REQUESTED** [review]

**Comment by saviourcode** [comment]
@Shubham-1105 check now

**Comment by Shubham-1105** [comment]
Yes

**Comment by saviourcode** [comment]
LGTM

**Comment by Shubham-1105** [comment]
done

**Comment by saviourcode** [comment]
@Shubham-1105 We have changed few stuffs in the project. Please feel free to contribute to other open issues and close this PR.

**Pull request closed without merging**